ansible/templates/hieradata.j2

```diff
@@ -3,6 +3,7 @@ timezone::locality: '{{ timezone.locality }}'
 cloudwatchlogs::region: '{{ aws.region }}'
 
 config::base::install_amazon_ssm_agent: {{ aws.install_ssm_agent }}
+config::base::install_cloudwatchlogs: {{ aws.install_cloudwatchlogs }}
```

Comment:
Please add inspec test for the existence of cloudwatch configuration file when install cloudwatch flag is set to true.

Reply:
Done